stapi/display/display.tcl

```diff
@@ -229,14 +229,19 @@ catch { ::itcl::delete class STDisplay }
 		} else {
 			# TODO: this is not very good; should also hex-encode many other things.
 			foreach \
-				src " &		   {\"}		 <       > " \
+				src " &	       {\"}      <       > " \
```

Comment:
This only shows up because it had mixed spaces and tabs and it was hard to figure out WTF was going on.